aretomo/protocols/protocol_aretomo.py

```diff
@@ -457,19 +463,33 @@ def runAreTomoStep(self, tsId: str, tsFn: str):
             self.error('Aretomo execution failed for tsId %s -> %s' % (tsId, e))
 
     def createOutputStep(self, tsId: str, tsFn: str):
-        if tsId in self._failedTsList:
-            self.createOutputFailedTs(tsId)
-        else:
-            self.createOutputTs(tsId, tsFn)
+        with self._lock:
+            if tsId in self._failedTsList:
+                self.createOutputFailedTs(tsId)
+            else:
+                self.createOutputTs(tsId, tsFn)
+            for outputName in self._possibleOutputs.keys():
+                output = getattr(self, outputName, None)
+                if output:
+                    output.close()
 
     def createOutputTs(self, tsId: str, tsFn: str):
         self.info(f'------- createOutputStep ts_id: {tsId}')
 
         ts = self.getTsFromTsId(tsId)
         extraPrefix = self._getExtraPath(tsId)
+        AretomoAln = readAlnFile(self.getFilePath(tsFn, extraPrefix, ".aln"))
```

Comment:
if you skip alignment and do only reconstruction, this file does not exist,no?

Reply:
You're right. I'll write a test considering that case and go properly through it. Thanks